server/statistics/hot_peer_cache.go

```diff
@@ -385,11 +385,11 @@ func (f *hotPeerCache) updateHotPeerStat(newItem, oldItem *HotPeerStat, bytes, k
 		if interval == 0 {
 			return nil
 		}
+		isHot := bytes/interval.Seconds() >= newItem.thresholds[byteDim] || keys/interval.Seconds() >= newItem.thresholds[keyDim]
+		if !isHot {
+			return nil
+		}
 		if interval.Seconds() >= RegionHeartBeatReportInterval {
-			isHot := bytes/interval.Seconds() >= newItem.thresholds[byteDim] || keys/interval.Seconds() >= newItem.thresholds[keyDim]
-			if !isHot {
-				return nil
-			}
```

Comment:
I'm wondering what's difference here? It seems the result won't be changed with the given `isHot` and `interval.Seconds() >= RegionHeartBeatReportInterval`

Reply:
it will directly return nil when the interval is less than RegionHeartBeatReportInterval